src/views/pacbio/PacbioSampleIndex.vue

```diff
@@ -175,6 +187,9 @@ export default {
     async fetchRequests() {
       return await this.fetchWithQueryParams(this.setRequests, this.filterOptions)
     },
+    async updateLocations(locationsData) {
```

Comment:
You are doing this in all of the components. Some of them are asynchronous and others aren't.

I am wondering if based on that they should all be the same and maybe should they be a composable? @seenanair or @BenTopping can you advise.